packages/manager/modules/netapp/src/routing.js

```diff
@@ -50,6 +50,29 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
       schema: /* @ngInject */ ($http) =>
         $http.get('/storage.json').then(({ data }) => data),
+
+      topbarOptions: /* @ngInject */ ($translate, goToOrder) => ({
+        cta: {
+          type: 'button',
+          displayed: true,
+          disabled: false,
+          label: $translate.instant('netapp_order_cta_label'),
+          value: $translate.instant('netapp_order_cta_value'),
+          onClick: () => {
+            goToOrder();
+          },
+        },
+      }),
```

Comment:
**question**:

Maybe this logic could be set directly in `ListLayoutHelper.stateResolves` no? It looks like it can be common to other product? And by this let the user define only `goToOrder` in the resolve

Reply:
Yes sure it could be great.

If i understand well the ```ListLayoutHelper.stateResolves``` is here to define some common configuration.
So, if i define the topbarOptions into `stateResolves` i will have to redefine it in product routing because i have to define the label and value content and the onClick behavior which could be different for each product. I think we have to let implementation into product.

Let me know if i can improve it.